src/i18n/strings/en_EN.json

```diff
@@ -805,7 +805,6 @@
     "%(brand)s is experimental on a mobile web browser. For a better experience and the latest features, use our free native app.": "%(brand)s is experimental on a mobile web browser. For a better experience and the latest features, use our free native app.",
     "Use app": "Use app",
     "Your homeserver has exceeded its user limit.": "Your homeserver has exceeded its user limit.",
-    "This homeserver has been blocked by it's administrator.": "This homeserver has been blocked by it's administrator.",
```

Comment:
This got "removed" because the correct spelling already existed in another place of the code.